modules/fedimint-ln-server/tests/ln_contracts.rs

```diff
@@ -31,25 +35,22 @@ async fn test_outgoing() {
     .unwrap();
 
     let ctx = secp256k1::Secp256k1::new();
-    let gw_pk = KeyPair::new(&ctx, &mut rng).x_only_public_key().0;
+    let kp = KeyPair::new(&ctx, &mut OsRng);
+    let gw_pk = PublicKey::from_keypair(&kp).x_only_public_key().0;
+    let sec_key = SecretKey::from_keypair(&kp);
     let user_pk = KeyPair::new(&ctx, &mut rng).x_only_public_key().0;
     let preimage = Preimage([42u8; 32]);
     let hash = secp256k1::hashes::sha256::Hash::hash(&preimage.0);
 
-    let invoice: lightning_invoice::Invoice =
```

Comment:
Using an old invoice will cause an expired invoice error in [apply_output](https://github.com/fedimint/fedimint/pull/1074/commits/1ea20954bb029c09c36cf2febdcf621f064a5bac#diff-d777522b5e9ff491a0359a5506622730cabc67755d33cfab1a5ebf3cf1b0738dR506-R514)

I fixed it by creating an invoice with InvoiceBuilder which will have the default expiry based on the current time